docs/configuration/modules/load-flow-action-simulator.md

```diff
@@ -23,9 +23,9 @@ modify the structure of the network.
 Set the `ignore-pre-contingency-violations` to `true` to ignore the the pre-contingency violations and continue the
 simulation even if there are still violations after the pre-contingency simulation.
 
-## load-flow-factory
-Use the `load-flow-factory` property to define the `com.powsybl.loadflow.LoadFlowFactory` implementation to use for the
-simulation.
+## load-flow-name
+The `load-flow-name` property is an optional property that defines the implementation name to use for running the loadflow. 
+If this property is not set, the default loadflow implementation is used.
```

Comment:
You should explain (or add a reference) on what happens if no implementation or several implementation are found.